security-utils/src/main/java/com/yahoo/security/tls/ConfigFiledBasedTlsContext.java

```diff
@@ -31,20 +31,21 @@
 import java.util.logging.Logger;
 
 /**
- * A {@link TlsContext} that regularly reloads the credentials referred to from the transport security options file.
+ * A {@link TlsContext} that uses the tls configuration specified in the transport security options file.
+ * The credentials are regularly reloaded to support short-lived certificates.
  *
  * @author bjorncs
  */
-public class ReloadingTlsContext implements TlsContext {
+public class ConfigFiledBasedTlsContext implements TlsContext {
```

Comment:
Shouldn't it be `ConfigFileBasedTlsContext`?

Reply:
Yes, that's a typo. Will fix :)

Reply:
Done